apps/api/src/routes/schema.ts

```diff
@@ -56,6 +56,29 @@ export const keySchema = z
         "The number of requests that can be made with this key before it becomes invalid. If this field is null or undefined, the key has no request limit.",
       example: 1000,
     }),
+    refill: z
+      .object({
+        refillInterval: z.enum(["null", "daily", "monthly"]).openapi({
+          description: "Determines the rate verifications will be refilled.",
+          example: "daily",
+        }),
+        refillIncrement: z.number().int().openapi({
+          description: "How many verifications to refill during each refillInterval.",
+          example: 100,
+        }),
+        lastRefillAt: z.number().optional().openapi({
+          description: "The unix timestamp in miliseconds when the key was last refilled.",
+          example: 100,
+        }),
+      })
+      .optional()
+      .openapi({
+        description: "Unkey allows you to replenish verifications on a key on a refular interval.",
+        example: {
+          refillInterval: "daily",
+          refillIncrement: 10,
```

Comment:
these are already nested inside a `refill` block, right?, so we can remove the prefixes here.
also `increment` still sounds like a relative addition, which it isn't anymore :)